python/lib/dcm2bids_imaging_pipeline_lib/push_imaging_files_to_s3_pipeline.py

```diff
@@ -262,8 +262,7 @@ def _clean_up_empty_folders(self):
 
         # remove empty folders from file system
         print("Cleaning up empty folders")
-        cand_id = self.subject_id_dict["CandID"]
-        bids_cand_id = f"sub-{cand_id}"
+        bids_cand_id = f"sub-{self.subject_info.cand_id}"
         lib.utilities.remove_empty_folders(os.path.join(self.data_dir, "assembly_bids", bids_cand_id))
-        lib.utilities.remove_empty_folders(os.path.join(self.data_dir, "pic", cand_id))
+        lib.utilities.remove_empty_folders(os.path.join(self.data_dir, "pic", self.subject_info.cand_id))
```

Comment:
I get a type error here because self.subject_info.cand_id is an int now

```suggestion
        lib.utilities.remove_empty_folders(os.path.join(self.data_dir, "pic", str(self.subject_info.cand_id)))
```